app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -192,8 +190,8 @@ def parse_image(image, is_public)
       :vendor             => "amazon",
       :raw_power_state    => "never",
       :template           => true,
-      :labels             => labels,
-      :tags               => map_labels('Image', labels),
+      :labels             => parse_labels(image.tags),
+      :tags               => map_labels('Image', parse_tags(image.tags)),
```

Comment:
BTW, my original intent for "types" that ContainerLabelTagMapping deals with was model (base) names, essentially matching the custom_attribute `resource_type` polymorphic link.
And AFAICT there is no actual model called `Image`?

In practice though you can pass an arbitrary string to `map_labels()`, it doesn't even see the custom_attribute record, it will work as long as you defined mappings expecting this arbitrary string (which you do with `Amazon::Image` in MAPPABLE_ENTITIES).